### ISSUE TITLE
Accessibility: media queries should be in em, dimensions in rem

### ISSUE THREAD

**Issue by kaelig** [design-tokens/designtokens.org#4]
We should write CSS using px, but the compiled CSS should be in em/rem.

Something like running PostCSS at build time should do the trick.
